### DISCUSSION TITLE
upgrade codemirror

### DISCUSSION THREAD

**Discussion by fiammybe** [ImpressCMS/impresscms#985] [General]
We are currently (as far as I could find) using codemirror 0.62 from 2009 - it should be upgraded.

[a comment by an automated account is not shown]

**Comment by fiammybe** [comment]
Am I correct if I say that codemirror is used only in the HTML view of a content page?

**Reply by MekDrop** [reply]
probably yes; it for all object properties of source type, but I'm unsure how often this type is used.

**Comment by fiammybe** [comment]
I guess simply updating the script can be enough for the moment in those conditions. No need to rewrite that part 😉

**Comment by MekDrop** [comment]
Done for 2.0 as module -> https://github.com/ImpressCMS/codemirror-integration

**Reply by fiammybe** [reply]
I believe that means this can be closed.
